integration-test/src/test/java/org/sagebionetworks/ITDataAccessTest.java

```diff
@@ -861,12 +874,136 @@ public void testGetSubmission() throws SynapseException, JSONObjectAdapterExcept
+	}
+
+	@Test
+	public void testListUserSubmissionSearchResult() throws SynapseException, JSONObjectAdapterException {
+		SynapseClient synapseOne = new SynapseClientImpl();
+		userOneId = SynapseClientHelper.createUser(adminSynapse, synapseOne, true, true);
+		SynapseClient synapseTwo = new SynapseClientImpl();
+		userTwoId = SynapseClientHelper.createUser(adminSynapse, synapseTwo, true, true);
+
+		managedAR = new ManagedACTAccessRequirement()
+				.setAccessType(ACCESS_TYPE.DOWNLOAD)
+				.setSubjectIds(Collections.singletonList(new RestrictableObjectDescriptor().setId(project.getId()).setType(RestrictableObjectType.ENTITY)));
+
+		managedAR = adminSynapse.createAccessRequirement(managedAR);
+
+		ResearchProject rp = synapse.getResearchProjectForUpdate(managedAR.getId().toString());
+
+		rp.setInstitution("Sage");
+		rp.setProjectLead("Lead");
+		rp.setIntendedDataUseStatement("intendedDataUseStatement");
+		rp.setAccessRequirementId(managedAR.getId().toString());
+
+		rp = synapse.createOrUpdateResearchProject(rp);
+
+		RequestInterface request = synapse.createOrUpdateRequest(new Request()
+				.setResearchProjectId(rp.getId())
+				.setAccessRequirementId(managedAR.getId().toString())
+				.setAccessorChanges(Arrays.asList(
+						new AccessorChange().setType(AccessType.GAIN_ACCESS).setUserId(adminSynapse.getMyProfile().getOwnerId()),
+						new AccessorChange().setType(AccessType.GAIN_ACCESS).setUserId(synapse.getMyProfile().getOwnerId())
+				)));
+
+		SubmissionStatus submissionStatus = synapse.submitRequest(new CreateSubmissionRequest()
+				.setRequestId(request.getId())
+				.setRequestEtag(request.getEtag())
+				.setSubjectId(project.getId())
+				.setSubjectType(RestrictableObjectType.ENTITY));
+
+		submissionId = submissionStatus.getSubmissionId();
+
+		// Admin is able to fetch the submission
+		Submission submission = adminSynapse.getDataAccessSubmission(submissionStatus.getSubmissionId());
+
+		UserSubmissionSearchRequest userRequest = new UserSubmissionSearchRequest();
+
+		//call under test. SynapseOne is not accessor of Submission so empty result will be returned
+		UserSubmissionSearchResponse response = synapseOne.listUserSubmissionSearchResults(userRequest);
+
+		assertTrue(response.getResults().isEmpty());
+
+		// create approval for the requirement
+		AccessApproval approval = new AccessApproval();
+		approval.setAccessorId(synapse.getMyProfile().getOwnerId());
+		approval.setRequirementId(managedAR.getId());
+		approval.setRequirementVersion(managedAR.getVersionNumber());
+		approval.setSubmitterId(submission.getSubmittedBy());
+		AccessApproval expectedAccessApproval = adminSynapse.createAccessApproval(approval);
+
+		// call under test
+		UserSubmissionSearchResponse responseTwo = synapse.listUserSubmissionSearchResults(userRequest);
+
+		assertEquals(responseTwo.getResults().size(), 1);
+		assertEquals(responseTwo.getResults().get(0).getUserAccessApproval(), expectedAccessApproval);
```

Comment:
The expected is the first arg

Reply:
Also, assertEquals(List.of(new UserSubmissionSearchItem(expectedAcessApproval)), responseTwo.getResults())